client/blocks/importer/wordpress/upgrade-plan/upgrade-plan-hosting-details.tsx

```diff
@@ -77,11 +77,16 @@ export const UpgradePlanHostingDetails = () => {
 				<div className="import__upgrade-plan-hosting-details-header">
 					<p className="import__upgrade-plan-hosting-details-header-main">{ headerMainText }</p>
 					<p className="import__upgrade-plan-hosting-details-header-subtext">
-						{ translate( 'Check our performance, compared to the average WordPress host' ) }
+						{ translate(
+							'We offer a proven 24% boost* in Core Web Vitals—endorsed by data from Google.'
```

Comment:
The base for this calc is the CWM from April 2024. (62.1-38.31)

See more: pdxWSz-1pk-p2